utils/rpms/daos.spec

```diff
@@ -265,7 +265,6 @@ This is the package needed to manage server storage firmware on DAOS servers.
 %endif
 %{scons_exe} %{?_smp_mflags} \
       --config=force         \
-      --no-rpath             \
```

Comment:
Is this some previous attempt that needs to be reverted?  Ultimately, rpaths MUST be disabled for RPMs.

Reply:
I can't get the RPM to build without rpath, any ideas @brianjmurrell @jolivier23 ?

Reply:
fixed in most recent commit